.storybook/components/DesignTokens/Tier1/TypographyPresets.js

```diff
@@ -68,7 +69,7 @@ export class Tier1TypographyPresets extends Component {
       };
 
       return (
-        <>
+        <Grid key={`tier-1-typography-preset-${index}`}>
```

Comment:
Needed to use a `Grid` instead of a `div` here because there are 3 tokens in here and grouping them removed the gap between them. I also tried putting the key on each `TokenSpecimen`, but that didn't remove the warning.